ibikecph-lib/src/main/java/com/spoiledmilk/ibikecph/login/LoginSplashActivity.java

```diff
@@ -114,78 +114,78 @@ public void launchMainMapActivity() {
+                            finish();
+
+                        }
+                    });
+                } else {
+                    LoginSplashActivity.this.runOnUiThread(new Runnable() {
+                        @Override
+                        public void run() {
+                            findViewById(R.id.progressBar).setVisibility(View.GONE);
+                            launchMainMapActivity();
+                        }
+                    });
+                }
+            }
+        }).start();
     }
 
     public void onBtnLogInClick(View v) {
-//        Intent i = new Intent(this, LoginActivity.class);
-//        startActivityForResult(i, LOGIN_REQUEST);
+        Intent i = new Intent(this, LoginActivity.class);
+        startActivityForResult(i, LOGIN_REQUEST);
     }
 
     public void onEnableTrackingClick(View v) {
 
-//        if (IBikeApplication.getSignature().equals("")) {
-//            if (IBikeApplication.isFacebookLogin()) {
-//                Log.d("DV", "Prompting Facebookuser to create a password!");
-//                Intent i = new Intent(LoginSplashActivity.this, SignatureActivity.class).addFlags(Intent.FLAG_ACTIVITY_CLEAR_TOP);
-//                i.putExtra("fromSplashScreen", true);
-//                IBikeApplication.setWelcomeScreenSeen(true);
-//                startActivity(i);
-//            } else if (IBikeApplication.isUserLogedIn()) {
-//                Log.d("DV", "Prompting login for user!");
-//                Intent i = new Intent(LoginSplashActivity.this, SignatureActivity.class).putExtra("normalUser", true).addFlags(Intent.FLAG_ACTIVITY_CLEAR_TOP);
-//                IBikeApplication.setWelcomeScreenSeen(true);
-//                startActivity(i);
-//            }
-//        } else {
-//            Log.d("DV", "We got a signature, enabling tracking!");
-//            IBikeApplication.getSettings().setTrackingEnabled(true);
-//            launchMainMapActivity();
-//        }
+        if (IBikeApplication.getSignature().equals("")) {
+            if (IBikeApplication.isFacebookLogin()) {
+                Log.d("DV", "Prompting Facebookuser to create a password!");
+                Intent i = new Intent(LoginSplashActivity.this, SignatureActivity.class).addFlags(Intent.FLAG_ACTIVITY_CLEAR_TOP);
+                i.putExtra("fromSplashScreen", true);
+                IBikeApplication.setWelcomeScreenSeen(true);
+                startActivity(i);
+            } else if (IBikeApplication.isUserLogedIn()) {
+                Log.d("DV", "Prompting login for user!");
+                Intent i = new Intent(LoginSplashActivity.this, SignatureActivity.class).putExtra("normalUser", true).addFlags(Intent.FLAG_ACTIVITY_CLEAR_TOP);
+                IBikeApplication.setWelcomeScreenSeen(true);
+                startActivity(i);
+            }
+        } else {
+            Log.d("DV", "We got a signature, enabling tracking!");
+            IBikeApplication.getSettings().setTrackingEnabled(true);
```

Comment:
This part of the code is only executed, if tracking is enabled. As it is now, tracking is disabled.
Since we do not intend to use this functionality, all code related to tracking should be cleaned up at some point.

Reply:
we'll do it later